test/index.js

```diff
@@ -61,18 +61,18 @@ const DUMMY_PACKAGE_JSON = { name: 'dummy', version: '1.0.0' }
 const netlifyConfig = { build: {} }
 
 describe('preBuild()', () => {
-  test('fail build if the app has static html export in npm script', async () => {
-    await expect(
-      plugin.onPreBuild({
-        netlifyConfig: { build: { command: 'npm run build' } },
-        packageJson: { ...DUMMY_PACKAGE_JSON, scripts: { build: 'next export' } },
-        utils,
-        constants: { FUNCTIONS_SRC: 'out_functions' },
-      }),
-    ).rejects.toThrow(
-      `Static HTML export Next.js projects do not require this plugin. Check your project's build command for 'next export'.`,
-    )
-  })
+  // test('fail build if the app has static html export in npm script', async () => {
```

Comment:
Instead of removing those tests, should those be updated to check that the plugin does not throw, nor call `next-on-netlify` under those scenarios?

Reply:
see my reply above